engine/src/test/java/io/camunda/zeebe/engine/metrics/ProcessEngineMetricsTest.java

```diff
@@ -0,0 +1,172 @@
+import io.camunda.zeebe.protocol.record.value.BpmnElementType;
+import io.camunda.zeebe.protocol.record.value.IncidentRecordValue;
+import io.camunda.zeebe.test.util.record.RecordingExporter;
+import io.prometheus.client.CollectorRegistry;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Optional;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+public class ProcessEngineMetricsTest {
+
+  @ClassRule public static final EngineRule ENGINE = EngineRule.singlePartition();
+
+  private static final String DMN_RESOURCE = "/dmn/drg-force-user.dmn";
+  private static final String PROCESS_ID = "process";
+  private static final String TASK_ID = "task";
+
+  @Before
+  public void resetMetrics() {
+    ProcessEngineMetrics.EVALUATED_DMN_ELEMENTS.clear();
+    ProcessEngineMetrics.EXECUTED_INSTANCES.clear();
+  }
+
+  @Test
+  public void shouldIncreaseSuccessfullyEvaluatedDmnElements() {
+    ENGINE
+        .deployment()
+        .withXmlClasspathResource(DMN_RESOURCE)
+        .withXmlResource(
+            Bpmn.createExecutableProcess(PROCESS_ID)
+                .startEvent()
+                .businessRuleTask(
+                    TASK_ID,
+                    t -> t.zeebeCalledDecisionId("force_user").zeebeResultVariable("result"))
+                .endEvent()
+                .done())
+        .deploy();
+
+    final long processInstanceKey =
+        ENGINE
+            .processInstance()
+            .ofBpmnProcessId(PROCESS_ID)
+            .withVariables(Map.of("lightsaberColor", "blue", "height", 175))
+            .create();
+
+    final Optional<Record<IncidentRecordValue>> first =
+        RecordingExporter.incidentRecords(IncidentIntent.CREATED).findFirst();
+    RecordingExporter.processInstanceRecords(ProcessInstanceIntent.ELEMENT_COMPLETED)
+        .withProcessInstanceKey(processInstanceKey)
+        .withElementType(BpmnElementType.PROCESS)
+        .await();
+
+    assertThat(
+            metricValue(
+                "zeebe_executed_instances_total",
+                entry("organizationId", "null"),
+                entry("type", "ROOT_PROCESS_INSTANCE"),
+                entry("action", "activated"),
+                entry("partition", "1")))
+        .isNotNull()
+        .isEqualTo(1);
+
+    assertThat(
+            metricValue(
+                "zeebe_executed_instances_total",
+                entry("organizationId", "null"),
+                entry("type", "ROOT_PROCESS_INSTANCE"),
+                entry("action", "completed"),
+                entry("partition", "1")))
+        .isNotNull()
+        .isEqualTo(1);
+
+    assertThat(
+            metricValue(
+                "zeebe_evaluated_dmn_elements_total",
+                entry("organizationId", "null"),
+                entry("action", "evaluated_successfully"),
+                entry("partition", "1")))
+        .isNotNull()
+        .isEqualTo(2);
```

Comment:
🔧 This assertion (and the same assertion in the other test) could benefit from a `.describedAs()` to explain why 2 is expected.